src/plugins/ramps/README.md

```diff
@@ -0,0 +1,54 @@
+# Ramp Plugins
+
+This directory contains the new ramp plugin architecture for buy/sell integrations.
+
+## Overview
+
+The ramp plugin system provides a standardized interface for integrating fiat on/off ramp providers. Unlike the legacy fiat plugin system, ramp plugins:
+
+- Have a simpler, more focused API
+- Return quote objects that can be approved later
+- Use direct API access instead of UI wrappers
+
+## Plugin Interface
+
+### RampPlugin
+The main plugin interface with:
+- `pluginId`: Unique identifier
+- `rampInfo`: Display information (name, icon)
+- `fetchQuote()`: Returns an array of quotes for all supported payment types
+
+### RampQuoteResult
+Quote objects returned by plugins with:
+- Quote details (amounts, currencies, etc.)
+- `approveQuote()`: Executes the quote
+- `closeQuote()`: Cleanup function
+
+### RampApproveQuoteParams
+Parameters passed to `approveQuote()`:
+- `coreWallet`: The wallet to use for the transaction
+
+### RampPluginConfig
+Configuration passed to plugin factories:
+- `initOptions`: Provider-specific initialization options
+- `store`: Optional storage interface for persistent data
+- `makeUuid`: Optional UUID generator
+- `account`: EdgeAccount for wallet operations
+- `navigation`: Navigation object for scene transitions
+- `onLogEvent`: Analytics tracking function
+- `disklet`: Disklet for file operations and permissions
```

Comment:
Provide disklet or store but not both. Whichever one is more recent and recommended by @swansontec

Reply:
The store is correct, but don't just grab the core one. The store is meant to be scoped by pluginId, so we need a small wrapper to set pluginId -> storeId before passing to the plugins.